includes/Core/CLI/Reset_CLI_Command.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Site Kit Cache CLI Commands
+ *
+ * @package   Google\Site_Kit\Core\CLI
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\CLI;
+
+use Google\Site_Kit\Core\Util\Reset;
+
+/**
+ * Resets Site Kit Settings and Data 
+ *
+ * @since n.e.x.t
+ */
+class Reset_CLI_Command extends CLI_Command {
+
+	/**
+	 * Resets Site Kit Settings and Data
```

Comment:
Let's use the same description here as `Reset::all()`
```suggestion
	 * Deletes options, user stored options, transients and clears object cache for stored options.
```

Reply:
Updated.